File: 인범/2026_06_11/clothes.py
def solution(clothes):
    from collections import defaultdict
    mydict=defaultdict(list)
    
    for item in clothes:
        cloth,typ = item
        mydict[typ].append(cloth)
    
    total = 1
    for val in mydict.values():
        total *= (len(val)+1)
    
    
    return total-1

File: 인범/2026_06_11/clothes.java
import java.util.*;

class Solution {
    public int solution(String[][] clothes) {
        int answer = 1;
        HashMap<String, Integer> map = new HashMap<>();
        
        for(String[] cloth : clothes){
            String item = cloth[0];
            String type = cloth[1];
            map.put(type, map.getOrDefault(type,0) + 1);
        }
        
        for(Integer amount : map.values()){
            answer *= (amount +1);
        }
        
        // System.out.println("Map : " + map);
        return answer -1;
    }
}